db/schema.rb

```diff
@@ -30,6 +30,12 @@
     "revoked",
   ], force: :cascade
 
+  create_enum :slipvector_survey_status, [
+    "preparing",
+    "in_progress",
+    "complete",
+  ], force: :cascade
+
```

Comment:
This seems like a case of mistaken schema change. `slipvector_survey_status` isn't related to this PR, and I don't see the schema change to add the `description` column to `rooms`.

Reply:
Womp womp. Sorry, I think I added the wrong bits. Will fix.

Reply:
My bad! I have a long-running-secret-branch that I never rolled back the migrations on.